portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,91 @@
+              I build web apps, Android apps and I dabble in other fun tech stuff. I love playing video games
+              and hanging out with my friends. Check out some of the things I've built and feel free to get in touch.
+              We could be friends!
+            </p>
+          </span>
+        </div>
+      </div>
+      <div id="portfolio">
+        <h3 id="portfolio-header">Some things I've built</h3>
+        <div class="card-container top-margin">
+          <span class="card-image">
+            <img class="image" src="/images/yelpcamp.png">
+          </span>
+          <span class="container-text">
+            <h4 class="project-name">YelpCamp</h4>
+            <p class="project-description">
+              YelpCamp is a like Yelp but for campgrounds. Users can signup, post campgrounds and comment on them. 
+              It was built with Node.js, Express and MongoDB
+            </p>
+            <div class="project-links">
+              <a class="view-app-link" href="https://yelpcamp001.herokuapp.com/" target="_blank">View App</a>
+              <a class="view-repo-link" href="https://github.com/prosper007/YelpCamp" target="_blank">View App Repo on Github</a>
+            </div>
+          </span>
+        </div>
+
+        <div class="card-container top-margin">
+          <span class="card-image">
+            <img class="image" src="/images/react_notes.png">
+          </span>
+          <span class="container-text">
+            <h4 class="project-name">React Notes</h4>
+            <p class="project-description">
+              React Notes is basically a Google Keep knock off. Updates to notes are made in real time across all 
+              connected clients like in Google Docs! It was built with React and a Realtime Firebase Database.  
+            </p>
+            <div class="project-links">
+              <a class="view-app-link" href="http://prosper-react-notes.surge.sh/" target="_blank">View App</a>
+              <a class="view-repo-link" href="https://github.com/prosper007/react-notes" target="_blank">View App Repo on Github</a>
+            </div>
+          </span>
+        </div>
+
+        <div class="card-container top-margin">
+          <span class="card-image">
+            <img class="half-image" src="/images/quake_report1.jpg">
+            <img class="half-image" src="/images/quake_report2.jpg">
+          </span>
+          <span class="container-text">
+            <h4 class="project-name">Quake Report</h4>
+            <p class="project-description">
+              Quake Report is an Android app that pulls earthquake data from USGS API and displays it in a nice, simple layout.
+              Users can filter and sort the displayed earthquakes by magnitude and time of occurence.
+            </p>
+            <div class="project-links">
+              <a class="view-app-link" href="https://play.google.com/store/apps/details?id=com.prosper.android.quakereport" target="_blank">View App</a>
```

Comment:
The Google HTML style guide recommends wrapping long lines: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

Do you want to introduce newlines to break this up a bit? Something like this:

```
<a class="view-app-link"
    href="https://play.google.com/store/apps/details?id=com.prosper.android.quakereport"
    target="_blank">
  View App
</a>
```

(Same thing in the long lines below.)